CITATION.cff

```diff
@@ -0,0 +1,48 @@
+cff-version: 1.2.0
+message: "If you use or extend Galaxy in your published work, please cite this publication as below"
+authors:
+  - family-names: "Afgan"
+    given-names: "Enis"
+  - family-names: "Baker"
+    given-names: "Dannon"
+  - family-names: "Batut"
+    given-names: "Bérénice"
+  - family-names: "van den Beek"
+    given-names: "Marius"
+  - family-names: "Bouvier"
+    given-names: "Dave"
+  - family-names: "Čech"
+    given-names: "Martin"
+  - family-names: "Chilton"
+    given-names: "John"
+  - family-names: "Clements"
+    given-names: "Dave"
+  - family-names: "Coraor"
+    given-names: "Nate"
+  - family-names: "Grüning"
+    given-names: "Björn"
+  - family-names: "Guerler"
+    given-names: "Aysam"
+  - family-names: "Hillman-Jackson"
+    given-names: "Jennifer"
+  - family-names: "Hiltemann"
+    given-names: "Saskia"
+  - family-names: "Jalili"
+    given-names: "Vahid"
+  - family-names: "Rasche"
+    given-names: "Helena"
+  - family-names: "Soranzo"
+    given-names: "Nicola"
+  - family-names: "Goecks"
+    given-names: "Jeremy"
+  - family-names: "Taylor"
+    given-names: "James"
+  - family-names: "Nekrutenko"
+    given-names: "Anton"
+  - family-names: "Blankenberg"
+    given-names: "Daniel"
+title: "The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2018 update"
+version: 18.05
```

Comment:
This is a required attribute, unclear what should be there for us. For now I've added the release at the time of publication.